src/dialects/postgres/query-generator.js

```diff
@@ -295,14 +295,15 @@ export class PostgresQueryGenerator extends AbstractQueryGenerator {
     return super.handleSequelizeMethod.call(this, smth, tableName, factory, options, prepend);
   }
 
-  addColumnQuery(table, key, attribute) {
+  addColumnQuery(table, key, attribute, options) {
```

Comment:
```suggestion
  addColumnQuery(table, key, attribute, options = {}) {
```
Any reason why the options object is not defaulted to an empty object?

Reply:
I think I am doing it down below.

```js
options = options || {}
```

The reason for doing this (the above way) is that I am passing `options` explicitly as null in case the `ifNotExists` argument wasn't provided.

Reply:
See my other comment about this, same holds for this one as well